<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>352</width>
    <height>39</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="PyDMLabel" name="description_label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string/>
       </property>
       <property name="text">
        <string>${DESC}</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="PyDMByteIndicator" name="camera_status_indicator">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Acquisition status: &lt;span style=&quot;color:green&quot;&gt;green=idle&lt;/span&gt;, &lt;span style=&quot;color:orange&quot;&gt;yellow=acquiring&lt;/span&gt;, white=disconnected)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="alarmSensitiveContent" stdset="0">
        <bool>false</bool>
       </property>
       <property name="alarmSensitiveBorder" stdset="0">
        <bool>true</bool>
       </property>
       <property name="channel" stdset="0">
        <string>${PREFIX}cam1:Acquire</string>
       </property>
       <property name="onColor" stdset="0">
        <color>
         <red>255</red>
         <green>255</green>
         <blue>0</blue>
        </color>
       </property>
       <property name="offColor" stdset="0">
        <color>
         <red>0</red>
         <green>255</green>
         <blue>0</blue>
        </color>
       </property>
       <property name="showLabels" stdset="0">
        <bool>true</bool>
       </property>
       <property name="bigEndian" stdset="0">
        <bool>false</bool>
       </property>
       <property name="circles" stdset="0">
        <bool>false</bool>
       </property>
       <property name="numBits" stdset="0">
        <number>1</number>
       </property>
       <property name="shift" stdset="0">
        <number>0</number>
       </property>
       <property name="labels" stdset="0">
        <stringlist>
         <string></string>
        </stringlist>
       </property>
      </widget>
     </item>
     <item>
      <widget class="PyDMLabel" name="camera_status_label">
       <property name="toolTip">
        <string/>
       </property>
       <property name="PyDMToolTip" stdset="0">
        <string>${PREFIX}cam1:DetectorState_RBV</string>
       </property>
       <property name="channel" stdset="0">
        <string>${PREFIX}cam1:DetectorState_RBV</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="imageJ_button">
       <property name="toolTip">
        <string>Launch ImageJ with the EPICS AD Viewer plugin loaded for this camera</string>
       </property>
       <property name="text">
        <string>ImageJ</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="PyDMRelatedDisplayButton" name="camera_viewer_button">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string/>
       </property>
       <property name="text">
        <string>Viewer</string>
       </property>
       <property name="filenames" stdset="0">
        <stringlist>
         <string>sample_viewer.ui</string>
        </stringlist>
       </property>
       <property name="showIcon" stdset="0">
        <bool>false</bool>
       </property>
       <property name="macros" stdset="0">
        <stringlist>
         <string>PREFIX=${PREFIX},DESC=${DESC}</string>
        </stringlist>
       </property>
       <property name="openInNewWindow" stdset="0">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PyDMLabel</class>
   <extends>QLabel</extends>
   <header>pydm.widgets.label</header>
  </customwidget>
  <customwidget>
   <class>PyDMByteIndicator</class>
   <extends>QWidget</extends>
   <header>pydm.widgets.byte</header>
  </customwidget>
  <customwidget>
   <class>PyDMRelatedDisplayButton</class>
   <extends>QPushButton</extends>
   <header>pydm.widgets.related_display_button</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
